lib/servos/src/Servos.cpp

```diff
@@ -1,59 +1,48 @@
-/*
-// Si el comentario es "//*" significa anotacion personal que luego sera removida, no es un comentario.
+/* 
+This is the servo controller library. The following functions are capable to controling the angle of the servos with a certain velocity.
+It can also attach or detach the servos, in order to consume less energy.
 */
-//HOLA
-
-
 
+//
 #include <Arduino.h>
 #include <Servo.h>
 #include <servos.h>
 
 Servo servoRight;
 Servo servoLeft;
 
-float lastAngleRight = 0;
-float lastAngleLeft = 0;
+int lastAngleRight = 0;
+int lastAngleLeft = 0;
 
 unsigned long actualTime = 0;
 unsigned long lastTimeLeft = 0;
 unsigned long lastTimeRight = 0;
 
 unsigned long interval = 10;
 
-/*
-PONER EN INITS
-
-servoRight.attach(9); //*Poner pin servo der.
-servoLeft.attach(9);  //*Poner pin servo izq.
-
-servoRight.write(0);
-servoLeft.write(0);
-
-*/
 int velocity = 0;
 
+int _rightPin;
+int _leftPin;
```

Comment:
int variables should always be initialized on definition

```suggestion
int _rightPin = 0;
int _leftPin = 0;
```